cli.py

```diff
@@ -33,22 +29,22 @@ def create_user():
     raw_password = getpass.getpass("Password: ")
     email = input("Email: ")
 
-    with Session(engine) as session:
-        u = (
-            session.query(db.User)
-            .where(or_(db.User.username == username, db.User.email == email))
-            .first()
-        )
-        if u is not None:
-            if u.username == username:
-                raise click.ClickException(f'User "{username}" already exists.')
-            else:
-                raise click.ClickException(f'Email "{email}" already exists.')
+    session = q.get_session()
+    u = (
```

Comment:
For ease of reading code, change `u` to `existing_user` and `user` to `new_user`.